articles/api/authentication/api-authz/_get-token.md

```diff
@@ -390,6 +390,109 @@ This is the OAuth 2.0 grant that highly-trusted apps use to access an API. In th
 - [Executing the Resource Owner Password Grant](/api-auth/tutorials/password-grant)
 - [Multi-factor Authentication and Resource Owner Password](/api-auth/tutorials/multifactor-resource-owner-password)
 
+## Device Authorization Flow
+
+```http
+POST https://${account.namespace}/oauth/token
+Content-Type: application/x-www-form-urlencoded
+
+client_id=${account.clientId}&device_code=YOUR_DEVICE_CODE&grant_type=urn:ietf:params:oauth:grant-type:device_code
+```
+
+```shell
+curl --request POST \
+  --url 'https://${account.namespace}/oauth/token' \
+  --header 'content-type: application/x-www-form-urlencoded' \
+  --data 'client_id=${account.clientId}&device_code=YOUR_DEVICE_CODE&grant_type=urn:ietf:params:oauth:grant-type:device_code'
+```
+
+```javascript
+var request = require("request");
+
+var options = { method: 'POST',
+  url: 'https://${account.namespace}/oauth/token',
+  headers: { 'content-type': 'application/x-www-form-urlencoded' },
+  form:
+   { client_id: '${account.clientId}',
+     device_code: 'YOUR_DEVICE_CODE',
+     grant_type: 'urn:ietf:params:oauth:grant-type:device_code' }
+   };
+
+request(options, function (error, response, body) {
+  if (error) throw new Error(error);
+
+  console.log(body);
+});
+```
+
+> RESPONSE SAMPLE:
+
+```JSON
+HTTP/1.1 200 OK
+Content-Type: application/json
+{
+   "access_token": "eyJz93a...k4laUWw",
+   "id_token": "eyJ...0NE",
+   "refresh_token": "eyJ...MoQ",
+   "scope": "...",
+   "expires_in": 86400,
+   "token_type": "Bearer"
+}
+```
+
+```JSON
+HTTP/1.1 400 BAD REQUEST
+Content-Type: application/json
+ { 
+  "error": "authorization_pending",
+  "error_description": "Retryable: We are waiting for the user to take some action; try polling again using the suggested interval."
+ }
+```
+
+```JSON
+HTTP/1.1 400 BAD REQUEST
+Content-Type: application/json
+ { 
+  "error": "slow_down",
+  "error_description": "Retryable: You are polling too fast; slow down and use the suggested interval."
+ }
+```
+
+```JSON
+HTTP/1.1 400 BAD REQUEST
+Content-Type: application/json
+ { 
+    "error": <anything else>,
+    "description": "Failure: User cancelled the confirmation prompt or consent page; the code expired; there was an error."
+ }
+```
+
+<%= include('../../../_includes/_http-method', {
+  "http_badge": "badge-success",
+  "http_method": "POST",
+  "path": "/oauth/token",
+  "link": "#device-auth"
+}) %>
+
+This is the OAuth 2.0 grant that input-constrained devices use to access an API. Poll this endpoint using the interval returned with your [device code](/api/authentication?http#device-code) to directly request an Access Token using the application's credentials (a Client ID) and a device code.
```

Comment:
Is this link right? it didn't show me anything

Reply:
So... intriguingly, this is a piece of functionality that seems to try to encase the GET label in an <a> tag with the `href=` being this "link". But the functionality obviously doesn't work on any of the endpoints. It isn't breaking anything either, though, so for now, we will leave this as is until we dig into this functionality further.

Reply:
Just to see if I understood, you want to merge it and fix it later or you want to leave the conversation unresolved until you dig into the issue?

Reply:
I misunderstood this issue. I was focusing on the badge code in the "include", but I see that you mean the link below that (just above your comment). Will be fixed in the next push.